python/ray/tune/config_parser.py

```diff
@@ -73,7 +73,7 @@ def make_parser(parser_creator=None, **kwargs):
         help="Whether to checkpoint at the end of the experiment. "
         "Default is False.")
     parser.add_argument(
-        "--no-sync-on-checkpoint",
+        "--sync-on-checkpoint",
         action="store_true",
         help="Disable sync-down of trial checkpoint, which is enabled by "
```

Comment:
Can you update this help string too? Thanks for the PR!

Reply:
I've updated the help string. And no problem!